source/keyboardHandler.py

```diff
@@ -203,7 +203,8 @@ def internal_keyDownEvent(vkCode, scanCode, extended, injected):
 			or (
 				keyCode == lastNVDAModifier
 				and lastNVDAModifierReleaseTime
-				and time.time() - lastNVDAModifierReleaseTime < config.conf["keyboard"]["multiPressTimeout"] / 1000
+				and time.time() - lastNVDAModifierReleaseTime
```

Comment:
Here, Ruff has made something less readable.

Thanks @SaschaCowley to have made it more readable again in aed380fa73930e5a569440850db83f1a4cd20205.

Anyway, can we do something to avoid this in the future?

Also cc @seanbudd